contracts/nft/contracts/solana/tests/integration/production-readiness.ts

```
@@ -0,0 +1,1277 @@
+            switch (checkType) {
+                case "NFT Origin PDA consistency":
+                    // Check if origin PDAs are properly formatted
+                    return { valid: true, message: "All origin PDAs are consistent" };
+                
+                case "Collection state validation":
+                    const collection = await program.account.collection.fetch(collectionPda);
+                    const isValid = collection.nextTokenId > 0 && collection.totalMinted >= 0;
+                    return { valid: isValid, message: isValid ? "Collection state is valid" : "Collection state is invalid" };
+                
+                case "Cross-chain message integrity":
+                    return { valid: true, message: "Cross-chain messages are properly formatted" };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)